libyul/optimiser/LoadResolver.cpp

```diff
@@ -56,13 +56,15 @@ void LoadResolver::visit(Expression& _e)
 {
 	DataFlowAnalyzer::visit(_e);
 
-	if (FunctionCall const* funCall = std::get_if<FunctionCall>(&_e))
+	if (FunctionCall const* funCall = std::get_if<FunctionCall>(&_e);
+		funCall && std::holds_alternative<BuiltinName>(funCall->functionName))
```

Comment:
This is a really annoying way to format a condition because the second line at first looks as if it was the code inside the `if` :)

```suggestion
	if (
		FunctionCall const* funCall = std::get_if<FunctionCall>(&_e);
		funCall && std::holds_alternative<BuiltinName>(funCall->functionName)
	)
```

Reply:
oops :D sorry for that, fixed